service/backend/common/account.go

```diff
@@ -0,0 +1,32 @@
+package common
+
+import (
+	"sort"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+)
+
+// SortUnique deduplicates given slice of coins and sorts them by UTXO id in ascending order for consistency
+//
+// Per https://docs.avax.network/apis/avalanchego/apis/p-chain#platformgetutxos
+// and https://docs.avax.network/apis/avalanchego/apis/c-chain#avaxgetutxos
+// paginated getUTXOs calls may have duplicate UTXOs in different pages, this helper eliminates them.
+func SortUnique(coins []*types.Coin) []*types.Coin {
```

Comment:
I think this can be replaced with the new 1.18 features `slices.Sort` + `slices.Compact` (might need to implement some interfaces in `types.Coin`)

Reply:
When we started the work on this repo, it was only 1.17 compatible and we have written our changes with that. I think a golang 1.18+ refactoring can be done along with the rest of the code in a separate PR.